wdl2cwl/main_oop.py

```diff
@@ -718,7 +718,7 @@ def main() -> None:
     # write to a file in oop_cwl_files
     if args.output:
         with open(args.output, "w") as result:
-            result.write(str(Converter.load_wdl_tree(args.workflow)))
+            result.write(convert(args.workflow))
```

Comment:
So if `--output`/`-o` is missing, we don't do any conversion? :-)

Reply:
I had noticed this in the previous `main.py` too. Good opportunity to fix that. Should we make he `-o` mandatory, or by default print to stdout?

Reply:
not mandatory, no, but remove this https://github.com/kinow/wdl-cwl-translator/blob/7653428821859abd1cdcc03155e86a7e06f633a2/wdl2cwl/main_oop.py#L123

or maybe refactor so that a handle is passed to that function and the `main` function decides if it is a file handle or `sys.stdout`?